### PR TITLE
Update Spring Version

### PR DESCRIPTION
**Pull request by kelwinpa**

### CODE REVIEW

File: apps/acme-identity/src/main/java/com/example/acme/identity/SecurityConfiguration.java [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
 public class SecurityConfiguration {
 
+	@Value("${spring.security.oauth2.resourceserver.jwt.jwk-set-uri}")
+	private String jwkSetUri;
+
 	@Bean
-	public SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity httpSecurity) {
-		httpSecurity.oauth2ResourceServer()
-					.jwt()
-				.jwtAuthenticationConverter(new ReactiveJwtAuthenticationConverterAdapter(new UserNameJwtAuthenticationConverter()));
```
Comment (kvmw) [inline apps/acme-identity/src/main/java/com/example/acme/identity/SecurityConfiguration.java:16]: Why `UserNameJwtAuthenticationConverter` is not used anymore?

**Review by kvmw — COMMENTED** [review]
`SecuirtyConfiguration` is totally different from the old one. 
I understand you had to replace the deprecated methods but why other changes?

File: apps/acme-identity/src/main/java/com/example/acme/identity/SecurityConfiguration.java (hunk 1 above)
Comment (kelwinpa) [reply apps/acme-identity/src/main/java/com/example/acme/identity/SecurityConfiguration.java:16]: done

File: apps/acme-identity/src/test/java/com/example/acme/identity/WhoAmIControllerTests.java
Comment (kvmw) [inline apps/acme-identity/src/test/java/com/example/acme/identity/WhoAmIControllerTests.java:1]: Is this test green?  
The controller class is returning the principal name for both "userId" and "useName":
```
return Map.of(
  "userId", principal.getName(),
  "userName", principal.getName());
``` 

So I was wondering if this test has been green at all?

Comment (kelwinpa) [reply apps/acme-identity/src/test/java/com/example/acme/identity/WhoAmIControllerTests.java:1]: The test was never green, but I fix it now and it is now green

**Review by kvmw — APPROVED** [review]

**Pull request merged**